src/test/java/seedu/address/logic/commands/AddEventCommandTest.java

```diff
@@ -42,6 +43,16 @@ public void execute_eventAcceptedByModel_addSuccessful() throws Exception {
         assertEquals(Arrays.asList(validEvent), modelStub.eventsAdded);
     }
 
+    @Test
+    public void execute_duplicatePerson_throwsCommandException() {
```

Comment:
Typo in the method name

```suggestion
    public void execute_duplicateEvent_throwsCommandException() {
```